### DISCUSSION TITLE
Does o3de have a complete and runnable RPI sample code?

### DISCUSSION THREAD

**Discussion by cppupu1** [o3de/o3de#10956] [General]
Only code snippets can be found in the document。

**Comment by VickyAtAZ** [comment]
There is a o3de project AtomSampleViewer which used for demonstrate Atom Renderer. It includes RHI samples, RPI samples and samples for graphics features. 
The project is located here: https://github.com/o3de/o3de-atom-sampleviewer
You can find some documentations about the samples in this project: https://github.com/o3de/o3de-atom-sampleviewer/wiki.
